main/templates/main/base.html

```diff
@@ -11,17 +10,17 @@
     <meta name="description" content="drunc ui">
     <meta name="keywords" content="">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    {% bootstrap_css %}
-    {% bootstrap_javascript %}
     {% block extra_css %}
     {% endblock extra_css %}
     {% block extra_js %}
     {% endblock extra_js %}
     <script src="{% static 'main/js/htmx.min.js' %}"></script>
     <script src="{% static 'main/js/_hyperscript.min.js' %}"></script>
+    <script src="{% static 'main/js/bootstrap.bundle.min.js' %}"></script>
     <link rel="shortcut icon"
           type="image/png"
           href="{% static 'main/images/Favicon.png' %}">
+    <link rel="stylesheet" href="{% static 'main/css/bootstrap.min.css' %}">
```

Comment:
So, the theme is set by using this CSS file, not by setting any flag or configuration setting to `sandstone` anywhere?

Reply:
Yeah on the bootswatch website you can get the css for each theme